packages/minifiers-css/minifiers.js

```diff
@@ -121,17 +122,39 @@ var rewriteRules = function (rules) {
       var cssUrlRegex = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/gi;
       while (parts = cssUrlRegex.exec(value)) {
         oldCssUrl = parts[0];
-        quotes = parts[1];
+        quote = parts[1];
         resource = url.parse(parts[2]);
 
-        // Rewrite relative paths to absolute paths.
-        // We don't rewrite urls starting with a protocol definition such as
+
+        // We don't rewrite URLs starting with a protocol definition such as
         // http, https, or data.
-        if (isRelative(resource.path) && resource.protocol === null) {
+        if (resource.protocol !== null) {
+          continue;
+        }
+
+        // Rewrite relative paths (that refers to the internal application tree)
+        // to absolute paths (addressable from the public build).
+        if (isRelative(resource.path)) {
           absolutePath = pathJoin(basePath, resource.path);
-          newCssUrl = "url(" + quotes + absolutePath + quotes + ")";
-          value = value.replace(oldCssUrl, newCssUrl);
+        } else {
+          absolutePath = resource.path;
         }
+
+        // We used to finish the rewriting process at the absolute path step
+        // above. But it didn't work in case the Meteor application was deployed
+        // under a sub-path (eg `ROOT_URL=http://localhost:3000/myapp meteor`)
+        // in which case the resources linked in the merged CSS file would miss
+        // the `myapp/` prefix. Since this path prefix is only known at launch
+        // time (rather than build time) we can't use absolute paths to link
+        // resources in the generated CSS.
+        //
+        // Instead we transform absolute paths to make them relative to the
+        // merged CSS, leaving to the browser the responsibility to calculate
+        // the final resource links (by adding the application deployment
+        // prefix, here `myapp/`, if applicable).
+        relativeToMergedCss = path.relative(mergedCssPath, absolutePath);
```

Comment:
is this using the node version of `path`? Can you check that it will work on Windows? For safety, I recommend to change it to `files.pathRelative(...`